message_definitions/v1.0/ras_a.xml

```diff
@@ -0,0 +1,124 @@
+        <param index="2" label="X Position in the camera frame" minValue="0" increment="1">Position given by the number of pixels in X direction of the camera frame. Origin on the bottom left corner of the camera frame.</param>
+        <param index="3" label="Y Position in the camera frame" minValue="0" increment="1">Position given by the number of pixels in Y direction of the camera frame. Origin on the bottom left corner of the camera frame.</param>
+      </entry>
+    </enum>
+    <enum name="POI_GEOMETRY_TYPE">
+      <description>Point-of-Interest geometry types.</description>
+      <entry value="0" name="POI_GEOMETRY_TYPE_UNKNOWN">
+        <description>Unknown geometry.</description>
+      </entry>
+      <entry value="1" name="POI_GEOMETRY_TYPE_SQUARE">
+        <description>Square.</description>
+      </entry>
+      <entry value="2" name="POI_GEOMETRY_TYPE_RECTANGLE">
+        <description>Rectangle.</description>
+      </entry>
+      <entry value="3" name="POI_GEOMETRY_TYPE_CIRCLE">
+        <description>Circle.</description>
+      </entry>
+      <entry value="4" name="POI_GEOMETRY_TYPE_CUBE">
+        <description>Cube.</description>
+      </entry>
+      <entry value="5" name="POI_GEOMETRY_TYPE_CUBOID">
+        <description>Cuboid.</description>
+      </entry>
+      <entry value="6" name="POI_GEOMETRY_TYPE_SPHERE">
+        <description>Sphere.</description>
+      </entry>
+      <entry value="7" name="POI_GEOMETRY_TYPE_CYLINDER">
+        <description>Cylinder.</description>
+      </entry>
+    </enum>
+    <enum name="POI_CLASSIFICATION">
+      <description>
+        Point-of-Interest classification.
+        Provides default classes (0 to 2147483647 reserved), but implementation-specific classes can be considered instead on the field where this classification is set.
+      </description>
+      <entry value="0" name="POI_CLASSIFICATION_UNKNOWN">
+        <description>Unknown classification.</description>
+      </entry>
+      <entry value="1" name="POI_CLASSIFICATION_GEO_MARKER">
+        <description>Geolocation marker.</description>
+      </entry>
+      <entry value="2" name="POI_CLASSIFICATION_LANDING_PAD">
+        <description>Landing pad.</description>
+      </entry>
+      <entry value="3" name="POI_CLASSIFICATION_CLEAR_LANDING_SPOT">
+        <description>Detected suited landing spot.</description>
+      </entry>
+      <entry value="4" name="POI_CLASSIFICATION_BUILDING">
+        <description>Building.</description>
+      </entry>
+      <entry value="5" name="POI_CLASSIFICATION_PERSON">
+        <description>Person.</description>
+      </entry>
+      <entry value="6" name="POI_CLASSIFICATION_VEHICLE">
+        <description>Vehicle.</description>
+      </entry>
+      <entry value="7" name="POI_CLASSIFICATION_INGRESS_PORTAL">
```

Comment:
FMI What does this mean? I would guess you mean a hole you fly into to - like a window or door or under a bridge? If so, is it also and exit portal?

Reply:
> FMI What does this mean? I would guess you mean a hole you fly into to - like a window or door or under a bridge?

Window, door, hole, anything that allows entering a structure.

> If so, is it also and exit portal?

Good point. Maybe add an `POI_CLASSIFICATION_EGRESS_PORTAL`?